README.md

```diff
@@ -99,7 +99,7 @@ a disk named /dev/sda, you would run the following command to partition, format
 and mount the disk.
 
 ```console
-sudo nix --experimental-features "nix-command flakes" run github:nix-community/disko/latest -- --mode disko /tmp/disk-config.nix
```

Comment:
I figured most new people where quickly have flakes enabled anyway and the other ones know there way around.

Reply:
I don't know that that's true. Maybe a short note that you might have to add the extra experimental commands would be good for first-time users. This is the sort of stuff that makes the nix ecosystem super frustrating to get into: see command in docs, copypaste, doesn't work.

Reply:
You get this error message:

bash-5.2# nix run github:nix-community/disko/latest -- --mode disko /tmp/disk-config.nix
error: experimental Nix feature 'nix-command' is disabled; add '--extra-experimental-features nix-command' to enable it